debian/assets/carlos_ctl/config.py

```
@@ -0,0 +1,227 @@
+            die(f"CARLOS_PROVINCE ('{self.province}') must be 'on' or 'bc'")
+
+    @property
+    def flyway_locations(self) -> str:
+        return f"classpath:db/migration/common,classpath:db/migration/{self.province}"
+
+
+def load() -> Settings:
+    return Settings()
+
+
+def cmd_init_config(argv) -> int:
```

Ruff (0.16.1)

[warning] Too many branches (17 > 12)

(PLR0912)